src/test/scala/uk/gov/homeoffice/cirium/CiriumStreamToPortResponseSpec.scala

```
@@ -104,23 +113,22 @@ class CiriumStreamToPortResponseSpec extends TestKit(ActorSystem("testActorSyste
   "Given a stream of messages, each should end up in the correct port" >> {
 
     val client = new MockClient("https://latest")
-    val feed = Cirium.Feed(client, pollEveryMillis = 100)
+    val feed = Cirium.Feed(client, pollEveryMillis = 100, MockBackwardsStrategy("https://item/1"))
     val probe = TestProbe()
 
-    implicit val mat: ActorMaterializer = ActorMaterializer()
+    implicit val mat: Materializer = Materializer.createMaterializer(system)
 
     val flightStatusActor: ActorRef = system
       .actorOf(CiriumFlightStatusRouterActor.props(Map("TST" -> probe.ref)), "flight-status-actor")
 
-    feed.start(1, 2).map { source =>
-      source.runWith(Sink.actorRef(flightStatusActor, "complete"))
+    feed.start(2).map { source =>
+      source.runWith(Sink.actorRef(flightStatusActor, "complete", t => println(s"Failed with $t")))
```

Do you want log the failure instead of println ?